### PR TITLE
chore(alerts): deprecation notice for created_at/updated_at

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: website/docs/r/alert_policy.html.markdown [hunk 1]
```diff
@@ -33,8 +33,8 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
   * `id` - The ID of the policy.
-  * `created_at` - The time the policy was created.
-  * `updated_at` - The time the policy was last updated.
+  * `created_at` - **DEPRECATED** The time the policy was created.
```
Comment (ctrombley) [inline website/docs/r/alert_policy.html.markdown:36]: Add colons after DEPRECATED for consistency with @sanderblue 's PR?

https://github.com/terraform-providers/terraform-provider-newrelic/pull/505

Comment (sanderblue) [reply website/docs/r/alert_policy.html.markdown:36]: Yep, I think we're moving forward **DEPRECATED:**

Comment (zlesnr) [reply website/docs/r/alert_policy.html.markdown:36]: Can do.

Comment (zlesnr) [reply website/docs/r/alert_policy.html.markdown]: Done.

**Pull request merged**